azure-pipeline.yml

```diff
@@ -98,6 +100,11 @@ stages:
             -prepareMachine
             $(_BuildArgs)
           displayName: Windows Build / Publish
+        - task: PowerShell@2
+          inputs:
```

Comment:
I think this powershell task need to be before the build/publish task coz the build/publish task is the one that signs files. Did you test if the package created here is being signed?

Reply:
Oh - so build/publish does build/publish/sign? How do I hook into that?

Reply:
You just need to put the package in the folder that it picks for signing from, so basically putting your powershell task before the build/publish should sign your package along with others. you can verify that when its running your PR.